### PR TITLE
Add Dockerfile for NIKSS

### LINKED ISSUE #69: Create Dockerfile

### PR DESCRIPTION
**Pull request by osinstom**
Closes #69.

### CODE REVIEW

File: scripts/docker-build.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  if [ -e build ]; then /bin/rm -rf build; fi
+  mkdir -p build
+  cd build
+  cmake "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" ..
+  make -j$(nproc)
+  make install
+}
+
+function build_bpftool() {
+  git clone --recurse-submodules https://github.com/libbpf/bpftool.git /tmp/bpftool
```
Comment (tatry) [inline scripts/docker-build.sh:18]: This build of bpftool is independent from kernel version?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Build Docker images
+
+on:
+  push:
+    branches: [ master ]
```
[the remark this answers is not in the document]
Comment (osinstom) [reply .github/workflows/docker.yml:5]: IMO It's a better idea to reference a SHA of p4c to have reproducible builds. We can later update p4c version by changing the SHA.

File: scripts/docker-build.sh (hunk 1 above)
Comment (osinstom) [reply scripts/docker-build.sh:18]: It looks so. We have never built bpftool for a specific kernel version. The bpftool repo doesn't say anything about it too.

File: scripts/docker-build.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+set -e  # Exit on error.
+set -x  # Make command execution verbose
+
+function build_nikss() {
+  git submodule update --init --recursive
+  ./build_libbpf.sh
+  if [ -e build ]; then /bin/rm -rf build; fi
+  mkdir -p build
+  cd build
+  cmake "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" ..
```
[the remark this answers is not in the document]
Comment (osinstom) [reply scripts/docker-build.sh:12]: We could do it in future, if needed

**Pull request merged**